pytest_tests/testsuites/network/test_config_changes.py

```diff
@@ -0,0 +1,151 @@
+    "EigenTrustAlpha": "eigentrust_alpha",
+    "InnerRingCandidateFee": "inner_ring_candidate_fee",
+    "WithdrawFee": "withdrawal_fee",
+    "HomomorphicHashingDisabled": "homomorphic_hashing_disabled",
+    "MaintenanceModeAllowed": "maintenance_mode_allowed",
+    "unknown_key": "unknown_key",
+}
+
+
+@allure.title("Network configuration changes via neofs-adm")
+@pytest.mark.network_config
+class TestNetworkConfigChange(ClusterTestBase):
+    @pytest.mark.parametrize(
+        "key, value",
+        [
+            ("MaxObjectSize", 1048576),
+            ("BasicIncomeRate", 50000000),
+            ("AuditFee", 5000),
+            ("EpochDuration", 480),
+            ("ContainerFee", 2000),
+            # ("ContainerAliasFee",),
+            ("EigenTrustIterations", 8),
+            ("EigenTrustAlpha", 0.2),
+            ("InnerRingCandidateFee", 5000000000),
+            ("WithdrawFee", 200000000),
+            ("HomomorphicHashingDisabled", True),
+            ("MaintenanceModeAllowed", True),
+        ],
+    )
+    def test_config_update_single_value(self, key: str, value: Union[str, int, bool]):
+        allure.dynamic.title(f"Set '{key}' to '{value}'")
+
+        with allure.step(f"Get original value of '{key}'"):
+            storage_node = self.cluster.storage_nodes[0]
+            net_info = get_netmap_netinfo(
+                wallet=storage_node.get_wallet_path(),
+                wallet_config=storage_node.get_wallet_config_path(),
+                endpoint=storage_node.get_rpc_endpoint(),
+                shell=storage_node.host.get_shell(),
+            )
+            original_value = net_info[CONFIG_KEYS_MAPPING[key]]
+
+        self._set_and_verify_config_keys(**{key: value})
+
+        with allure.step(
+            f"Change '{key}' back to original value: '{original_value}' via neofs-adm"
+        ):
+            self._set_and_verify_config_keys(**{key: original_value})
+
+    @allure.title("Multiple network config keys can be changed at once")
+    def test_config_update_multiple_values(self):
+        new_key_value_pairs = {
+            "MaxObjectSize": 1048576,
+            "BasicIncomeRate": 50000000,
+            "AuditFee": 5000,
+        }
+        original_key_value_pairs = {}
+
+        with allure.step(f"Get original values of keys: {new_key_value_pairs.keys()}"):
+            storage_node = self.cluster.storage_nodes[0]
+            net_info = get_netmap_netinfo(
+                wallet=storage_node.get_wallet_path(),
+                wallet_config=storage_node.get_wallet_config_path(),
+                endpoint=storage_node.get_rpc_endpoint(),
+                shell=storage_node.host.get_shell(),
+            )
+            for key in new_key_value_pairs.keys():
+                original_key_value_pairs[key] = net_info[CONFIG_KEYS_MAPPING[key]]
+
+        self._set_and_verify_config_keys(**new_key_value_pairs)
+
+        with allure.step("Change network config back to original state"):
```

Comment:
It seems to me that changing the config to the original state is better made as a fixture

Reply:
It is actually philosophical question here, I consider this action as a part of the test, not a teardown. And fixtures are usually something that don't really belong to a test. I can move it to a fixture, but then it will be dissappeared from the test body, and will be a part of teardown.

Reply:
> It is actually philosophical question here, I consider this action as a part of the test, not a teardown. And fixtures are usually something that don't really belong to a test. I can move it to a fixture, but then it will be dissappeared from the test body, and will be a part of teardown.

If any of the test steps above fail, the network config  will not go back. This may lead to incorrect execution of next tests.

Reply:
Fixed.